config/common.txt

```diff
@@ -21,6 +21,7 @@ libblas=*=*openblas
 numpy>=2
 scipy
 matplotlib
+camb
```

Comment:
I am fine adding this package.  @iparask , I saw that you added `camb` to the tiger3 pip packages in #55 , so you may want to revert that if we add it here to the common conda package list.